components/timezone/data.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
praise: i like this data.json plan

(worth moving into `data/` though? or calling it globaldata.json)

Reply:
`data/` gets overwritten